zebra-state/src/service/check.rs

```diff
@@ -375,23 +375,23 @@ where
 pub(crate) fn initial_contextual_validity(
```

Comment:
Optional: we may also want to rename `prepared` in `validate_and_commit_non_finalized()` to match this, or it may be enough that it has `non_finalized` in the method name like the methods in `NonFinalizedState`.